frictionless/frictionless.go

```diff
@@ -40,6 +40,29 @@ type DataPackage struct {
 	Licenses []DataLicense `json:"licenses,omitempty"`
 	// a list identifying the sources for this resource (optional)
 	Sources []DataSource `json:"sources,omitempty"`
+	// a timestamp indicated when the package was created
+	Created string `json:"created,omitempty"`
```

Comment:
worth alphabetising this list of properties to make it simple to check whether such-and-such a property exists?